src/main/scala/ScalatraBootstrap.scala

```diff
@@ -22,6 +23,7 @@ class ScalatraBootstrap extends LifeCycle {
     context.mount(new PingApi, "/api/ping/*")
     context.mount(new MassTransitStopImportApi, "/api/import/*")
     context.mount(new IntegrationApi(Digiroad2Context.massTransitStopService), "/api/integration/*")
+    context.mount(new ViiteIntegrationApi(Digiroad2Context.roadAddressService), "/viite/api/integration/*")

```

Comment:
The local url to test is "http://localhost:8080/viite/api/integration/road_address?municipality=235" so it seems to be correct.